content/docs/get-started/index.md

```diff
@@ -163,46 +163,24 @@ requirements:
 
 </details>
 
-<details>
+## Model prediction
 
-### Click to learn about model codification
-
-#### Why do it the MLEM way ?
-
-Saving models to files or loading them back into python objects may seem like a
-deceptively simple task at first. For example, `pickle` and `torch` libraries
-can serialize/deserialize model objects to/from files. However, MLEM adds some
-"special sauce" by inspecting the objects and serializing their metadata into
-`.mlem` files and intelligently using this later on. This metadata is necessary
-to reliably enable actions like packaging and serving of different models types
-down in various ways. MLEM allows us to automate a lot of the pain points we
-would hit later on in our ML workflow by codifying and managing this metadata
-about our models (or other objects) for us.
-
-</details>
```

Comment:
My take: I'd keep this section. It's a bit too abstract for my taste, so I can suggest putting an example here. Saying something like:
- we need to know there is predict and predict_proba to build a REST API interface. Same for input data schema.
- we need to know it's pandas and numpy to pip install requirements in containers on Heroku/AWS

I'd keep it - with examples it can be easier to understand why MLEM is powerful.
WDYT?

Reply:
> I'd keep this section 

If we want to present the GS along these lines (give more motivation) then we should incorporate the ideas into the main text. E.g. instead of "Before we explore everything that MLEM can do, you need to save a machine learning model with MLEM." ([here](https://mlem.ai/doc/get-started?tab=Command-Line#saving-your-model)) we can use an explanation similar to the text above.

But if we put it in a hidden section that comes after another hidden section, to me that signals that we already decided this is not really important for the GS -- and I would agree it's best to keep the GS actionable (not theoretical). Thus moved it to a longer-format doc (that's linked several times from this page).

> putting an example

If I understand correctly this example would be disconnected from the main story in the GS which complicates things. We could def. add that example in the UG though, as another way to [expand on it](https://github.com/iterative/mlem.ai/pull/265#discussion_r1047651762).

Reply:
I suggest to move it away from GS in my recent GS PR here: https://github.com/iterative/mlem.ai/pull/281
`user-guide/basic-concepts` seems to make sense to me

Reply:
So the change in this PR seems to point in the right direction. We have to merge one PR at a time so I'll focus on this one for now but I'm happy to solve any conflicts that arise in #281, unless yo guys think it's easier to do #281 first (lmk).